src/app/index.ts

```diff
@@ -331,17 +302,12 @@ module.exports = yo.extend({
     this.log(`      ${chalk.green('Congratulations!')} Your add-in has been created! Your next steps:\n`);
```

Comment:
For greater clarity, would be great to show the same type of (colorful) message here at the top of the "post-install hints" as we show when the project creation process begins (changing the word "Creating" to "Created" at the start of the sentence, of course).

![image](https://user-images.githubusercontent.com/3019529/56169530-4725c680-5f93-11e9-817c-c941e1a7e572.png)

You could incorporate this message into the existing post-install hints as (roughly) shown here (changing the word "Creating" to "Created" on the second line of text):

![image](https://user-images.githubusercontent.com/3019529/56169698-d4691b00-5f93-11e9-843a-7d0bb6a8166d.png)

![image](https://user-images.githubusercontent.com/3019529/56169682-c7e4c280-5f93-11e9-9cfc-2d00e32c66e8.png)

![image](https://user-images.githubusercontent.com/3019529/56169706-ddf28300-5f93-11e9-9285-62f885e35fb0.png)

![image](https://user-images.githubusercontent.com/3019529/56169719-e6e35480-5f93-11e9-9c5c-aa66d01e22f4.png)

Reply:
@kbrandl Not sure I follow - let's discuss offline

Reply:
@TCourtneyOwen This isn't critical feedback; I'm going to mark this conversation as Resolved.